src/pkg/cli/client/byoc/do/byoc.go

```diff
@@ -52,9 +52,13 @@ var (
 type ByocDo struct {
 	*byoc.ByocBaseClient
 
-	buildRepo string
-	client    *godo.Client
-	driver    *appPlatform.DoApp
+	buildRepo          string
+	client             *godo.Client
+	driver             *appPlatform.DoApp
+	lastCdAppID        string
+	lastCdDeploymentID string
+	lastCdEtag         types.ETag
```

Comment:
is there a mechanism which persists these across multiple cli invocations or are these values only used downstream from memory? i can't tell at a glance if all of the references are downstream of an assignment.

Reply:
could be useful perhaps, but for now this is just to catch the case when we invoke tail right after deploy. AWS does the same.

Reply:
In that case, I wonder if the name `last...` is misleading. Aren't these the _current_ values?

Reply:
Hmm that is true

Reply:
Opened https://github.com/DefangLabs/defang/pull/933 to try to clear up the confusion